### PR TITLE
Add district census segmentation

### PR DESCRIPTION
**Pull request by tramuntanal**
This PR extends the segmentation capabilities of this Decidim App with district segmentation.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,14 @@
+require "csv"
+
 module ApplicationHelper
+
+  def pinbal_municipio_select(f, include_blank: true)
+    opts= {}
+    opts[:prompt]= "Selecciona el teu municipi" unless include_blank
```
Comment (LauraJaimeC) [inline app/helpers/application_helper.rb:7]: this text should not be localized?

File: app/services/consell_mallorca_authorization_handler.rb [hunk 2]
```diff
@@ -8,9 +8,11 @@ class ConsellMallorcaAuthorizationHandler < Decidim::AuthorizationHandler
   attribute :document_number, String
   attribute :surname, String
   attribute :birthdate, String
+  attribute :pinbal_municipio, String
 
   validates :document_type, inclusion: { in: [:dni, :nie, :passport] }, presence: true
   validates :document_number, presence: true
+  validates :pinbal_municipio, presence: true#, if: -> { ask_municipality_to_citizen? }
```
Comment (LauraJaimeC) [inline app/services/consell_mallorca_authorization_handler.rb:15]: Is if used?

File: db/schema.rb [hunk 3]
```diff
@@ -566,6 +566,56 @@
...
+  end
+
+  create_table "decidim_dummy_resources_nested_dummy_resources", force: :cascade do |t|
+    t.jsonb "translatable_text"
+    t.string "title"
+    t.bigint "dummy_resource_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```
Comment (LauraJaimeC) [inline db/schema.rb:618]: Be careful with dummy tables!

**Review by LauraJaimeC — CHANGES_REQUESTED** [review]
Good job! Some details.

**Comment by tramuntanal** [comment]
Thank you for the review, changes applied

**Pull request merged**